diff -ur bless-0.3.1/src/Makefile bless-0.3.1.new/src/Makefile
--- bless-0.3.1/src/Makefile	2005-02-14 14:13:56.000000000 +0000
+++ bless-0.3.1.new/src/Makefile	2005-03-03 20:53:47.563879256 +0000
@@ -76,11 +76,11 @@
 	$(MCS) -t:library -r:nunit.framework -pkg:gtk-sharp $<
 
 install: bless.exe 
-	install -c -d $(PREFIX)/lib/bless/bin/
-	install -c bless.exe $(PREFIX)/lib/bless/bin/
+	install -c -d $(DESTDIR)$(PREFIX)/lib/bless/bin/
+	install -c bless.exe $(DESTDIR)$(PREFIX)/lib/bless/bin/
 	echo -e "#!/bin/sh\ncd $(PREFIX)/lib/bless/bin/; mono bless.exe \"\$$@\"" > bless
-	install -c -d $(PREFIX)/bin
-	install -c bless $(PREFIX)/bin
+	install -c -d $(DESTDIR)$(PREFIX)/bin
+	install -c bless $(DESTDIR)$(PREFIX)/bin
 	rm bless
 
 uninstall:
